README.md

```diff
@@ -3,6 +3,8 @@
 This repository includes backup and recovery utilities for
 [GitHub Enterprise Server][1].
 
+**UPDATE**: `backup-utils` now requires [GNU awk](https://www.gnu.org/software/gawk) and [moreutils](https://joeyh.name/code/moreutils).
```

Comment:
As this PR can now run fine without `parallel`, we should probably change the wording here to reflect this is only needed for the beta functionality.

Reply:
Fixed